spec/fixtures/model/force/lease/lease_domain.json

```diff
@@ -2,11 +2,11 @@
   "application_id": "a0o0P00000FFc2yQAD",
   "id": "a130U0000001n4wQAA",
   "lease_start_date": {"year": "2028", "month": "01", "day": "01"},
-  "monthly_parking_rent": 100,
-  "monthly_tenant_contribution": 800,
+  "monthly_parking_rent": "$100.00",
```

Comment:
maybe increase some of these sample currencies to >=$1,000.00 to catch the lack of commas issue.